src/components/Review/CommentsList/CommentsList.tsx

```diff
@@ -0,0 +1,97 @@
+  }
+
+  const handleLikeComment = (commentId: string) => {
+    console.log(`Liking comment with ID ${commentId}`)
+  }
+
+  const handleDislikeComment = (commentId: string) => {
+    console.log(`Disliking comment with ID ${commentId}`)
+  }
+
+  return (
+    <>
+      <ul className='flex gap-10 flex-col'>
+        {loadedComments.map((comment, index) => (
+          <li className={`pb-10 ${index === loadedComments.length - 1 ? '' : 'border-b border-solid border-primary'}`} key={comment.id}>
+            <div className="font-medium text-[24px] text-primary mt-2">
+              <span>{comment.firstName} {comment.lastName}</span>
+            </div>
+            <div className="font-medium text-[18px] text-primary mt-2 mb-6 flex items-center">
+              <div className='flex items-center gap-1'>
+                {[...Array(5)].map((_, index) => (
+                  <FaStar className={`w-6 h-6 ${index < comment.rating ? 'text-positive' : 'text-disabled'}`} key={index} />
+                ))}
+              </div>
+              <span className="font-medium text-[18px] text-primary ml-2">{comment.rating}/5</span>
+              <div className="inline-flex font-medium text-[18px] text-tertiary">
+                <div className='inline-flex relative ml-4'>
+                  <span className='ml-4'>{comment.date}</span>
+                  <div className="text-tertiary h-2 w-2 rounded-full bg-gray-400 absolute top-1/2 left-0 transform -translate-x-1/2 -translate-y-1/2"></div>
+                </div>
+                <span className='ml-2'>{comment.time}</span>
+              </div>
+            </div>
+            <p className={`rounded-[8px] px-4 py-[17px] mb-6 ${comment.isCurrentUserComment ? 'bg-brand-second' : 'bg-secondary'}`}>{comment.text}</p>
+            <div className="flex justify-between items-center">
+              {comment.isCurrentUserComment && (
+                <Button onClick={() => handleDeleteComment(comment.id)} className="rounded-[47px] py-4 px-6 bg-secondary font-medium text-[18px] text-primary">Delete my review</Button>
+              )}
+              <div className='flex gap-2 ml-auto'>
+                <Button onClick={() => handleLikeComment(comment.id)} className="rounded-[47px] bg-secondary w-[88px] text-tertiary font-medium flex items-center justify-center gap-2"><BiLike /><span>{comment.likes}</span></Button>
+                <Button onClick={() => handleDislikeComment(comment.id)} className="rounded-[47px] bg-secondary w-[88px] text-tertiary font-medium flex items-center justify-center gap-2"><BiDislike /><span> {comment.dislikes}</span></Button>
+              </div>
+            </div>
+          </li>
+        ))}
+      </ul>
+
+      {showLoadMore && (
+        <Button onClick={loadMoreComments} className='flex items-center justify-center rounded-[47px] w-[278px] ml-auto mr-auto bg-secondary font-medium text-[18px] text-primary'>Show more</Button>
+      )}
+    </>
+  )
+}
+
+export default CommentList
```

Comment:
nit: we usually go with the "Default Export (Export as function)" style for components in our project. Noticed here and in all the other components "Named Export (Export as const)" was used—any particular reason for the switch? Just want to make sure we're keeping it consistent across the board. 😊

Reply:
write it down in your notebook of ideas and someday we'll harmonize the project.